### PR TITLE
Adding global toggles for TAGS and Facility ACL

### PR DESCRIPTION
**Pull request by aethelwulffe**
Merged PR did not include global conditional statements.
-Requires review of functionality of the tags as well as the facility
access control.

### CODE REVIEW

**Comment by aethelwulffe** [comment]
I have not tested this at all yet.  This is a work in progress in the very short near term.

File: library/globals.inc.php [hunk 1]
```diff
@@ -2372,7 +2378,7 @@
 
     'SMTP_PASS' => array(
       xl('SMTP Password for Authentication'),
-      'pass',                           // data type
+      'text',                           // data type
```
Comment (teryhill) [inline library/globals.inc.php]: You are clobbering my last commit

File: library/globals.inc.php [hunk 2]
```diff
@@ -74,7 +74,8 @@
   $perl_bin_dir        = 'C:/xampp/perl/bin';
   $temporary_files_dir = 'C:/windows/temp';
   $backup_log_dir      = 'C:/windows/temp';
-} else {
+}
+else {
```
Comment (teryhill) [inline library/globals.inc.php]: You are clobbering my last commit

Comment (teryhill) [reply library/globals.inc.php]: also did you mean to add features back to the user menu . It has no items under it.

**Comment by teryhill** [comment]
So tell me how I test it?

File: library/globals.inc.php (hunk 2 above)
Comment (aethelwulffe) [reply library/globals.inc.php]: Boy, did I hose this up...

I think you test this by giving me a minute to make another go...this commit was crap.

File: library/globals.inc.php (hunk 1 above)
Comment (aethelwulffe) [reply library/globals.inc.php]: I will rebase and have another go...I missed this.

File: library/globals.inc.php (hunk 2 above)
Comment (aethelwulffe) [reply library/globals.inc.php]: No.  I suppose you mean in another PR.  I did not intend to re-add features.

Comment (aethelwulffe) [reply library/globals.inc.php]: features has items under it...
I don't understand.

Comment (teryhill) [reply library/globals.inc.php]: That section is for the user preferences under miscellaneous

Comment (teryhill) [reply library/globals.inc.php]: Don't worry about it . I will fix it after the demos get up

Comment (aethelwulffe) [reply library/globals.inc.php]: demo site is a mess.

**Comment by aethelwulffe** [comment]
I am testing this branch from ground zero.  Creating everything from scratch.  Your install isn't working as it should, the demo is hosed up, and my install did everything exactly how I expected.

**Comment by teryhill** [comment]
Things I have noticed. You set the patient file to have a facility of '1" we send out the code with a facility of "3" so there will never a facility numbered 1. on my system I have them as 4 and 5.

I have tried every combination I can think of to break my system but it will not.

**Comment by aethelwulffe** [comment]
Never noticed that...wonder why we do it... doesn't seem to set the default anyway.

**Comment by teryhill** [comment]
My entire patient_data table has 1 except for the patient I changes and he has a 5

**Comment by aethelwulffe** [comment]
By adding two patients normally after a clean setup and UI only configuration, I have a two patients, one of whom I set to Facility 3 (the renamed default) and the other patient has a 0. 
  Nothing is working since I killed the old DB, I snuffed the old branch where this was working, and I only have a copy of the code intended for BPUC to use.  

I guess we have to revert this completely again.  I will see if I can re-create the feature, maybe outside of the "plugin" system.

**Comment by aethelwulffe** [comment]
Closing this PR.  Putting in an issue to revert the implementation of this feature.
If we do it, I suppose we should do it just using the tags system.
I will do a write-up on how to use it, put in issues to take care of the form formatting issues (really screwy) and create tools to do mass-setup features that link facilities to tags.

**Pull request closed without merging**